content/en/install.md

```diff
@@ -50,43 +55,87 @@ Other options include:
   and the IEP interactive development environment; Supports Linux,
   Windows, and Mac.
 
+If you already have Python installed, you can
+[install SciPy with `pip`](#pip-install).
+For more advanced users who will need to install or upgrade regularly,
+you may prefer to use a dedicated package manager like [`uv`]
+(see [Installing with `uv`](#uv-install)),
+[Conda], or [`pixi`] (see [Installing from conda-forge](#conda-forge-install)).
```

Comment:
-1 for Pixi. Looking at how things are moving right now, uv is going to completely shadow such alternative solution.

Reply:
I think it is worthwhile to mention it, at least while uv cannot handle conda packages (https://github.com/astral-sh/uv/issues/1703). In my experience, it is nice to have one tool for everything, rather than having to use something else for non-PyPI packages.

Reply:
I left an upvote on the issue and pinged Charlie to get some visibility. He helped us gladly in the past with Ruff 🤞

Reply:
@zanieb 😃 since you asked. Thanks 🙏

Reply:
We're not actively working on support for installing things from Conda in uv, so I think it's reasonable to have an alternative suggestion. Whether you prefer to just suggest using `mamba` or `pixi` seems dependent on how important the "project" use-case is. My 2-cents is that project-focused users usually use PyPI while the Conda users are usually more "environment" focused — in which case a project-based approach offered by uv and pixi is less important.

Reply:
That's great to hear! I definitely think Pixi is great and that project-based workflows are better than maintaining imperative global environments :) I'm just not sure how common it is.

Reply:
I don't think it's constructive at all to say no to `pixi` because of `uv. Basically, `uv` is a very nice upgrade from `pip` & co, while `pixi is a very nice upgrade from `conda` & co. 

> I guess I am bucking the trend by being a conda & project-workflow enjoyer then :)

I don't think you are. Many maintainers I talk to are excited about Pixi, because it gives more reproducible workflows. And as soon as you include compiled code, it can give you reproducibility in a way that PyPI-only tooling simply cannot.